File: 1045.py
t = [float (i) for i in raw_input().split()]
a, b, c = float(t[0]), float(t[1]), float(t[2])
t.sort()
a = t[2]
b = t[1]
c = t[0]
if (a >= (b + c)):
	print "NAO FORMA TRIANGULO"
else:
	if ((a * a) == (b * b) + (c * c)):
		print "TRIANGULO RETANGULO"
	if ((a * a) > (b * b) + (c * c)):
		print "TRIANGULO OBTUSANGULO"
	if ((a * a) < (b * b) + (c * c)):
		print "TRIANGULO ACUTANGULO"
	if (a == b and a == c):
		print "TRIANGULO EQUILATERO"
	if (a == b and c != a or b == c and a != b or a == c and b != a):
		print "TRIANGULO ISOSCELES"

File: 1045.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		double vet[] = new double[3];
		double maior, a, b, c;
		vet[0] = teclado.nextDouble();
		vet[1] = teclado.nextDouble();
		vet[2] = teclado.nextDouble();
		Arrays.sort(vet);

		a = vet[2];
		b = vet[1];
		c = vet[0];

		if (a >= (b + c)) {
			System.out.printf("NAO FORMA TRIANGULO%n");
		} else {
			if ((a * a) == (b * b) + (c * c)) {
				System.out.printf("TRIANGULO RETANGULO%n");
			}
			if ((a * a) > (b * b) + (c * c)) {
				System.out.printf("TRIANGULO OBTUSANGULO%n");
			}
			if ((a * a) < (b * b) + (c * c)) {
				System.out.printf("TRIANGULO ACUTANGULO%n");
			}
			if (a == b && a == c) {
				System.out.printf("TRIANGULO EQUILATERO%n");
			}
			if (a == b && c != a || b == c && a != b || a == c && b != a) {
				System.out.printf("TRIANGULO ISOSCELES%n");
			}
		}
	}
}
